project/plugins.sbt

```diff
@@ -3,14 +3,21 @@ resolvers += Resolver.url("HMRC-open-artefacts-ivy", url("https://open.artefacts
   Resolver.ivyStylePatterns
 )
 
+// To resolve a bug with version 2.x.x of the scoverage plugin - https://github.com/sbt/sbt/issues/6997
+// Try to remove when sbt 1.8.0+ and scoverage is 2.0.7+
+
+ThisBuild / libraryDependencySchemes ++= Seq(
+  "org.scala-lang.modules" %% "scala-xml" % VersionScheme.Always
+)
+
 addSbtPlugin("com.typesafe.play"         % "sbt-plugin"             % "2.8.18")
 addSbtPlugin("com.typesafe.sbt"          % "sbt-digest"             % "1.1.4")
 addSbtPlugin("net.ground5hark.sbt"       % "sbt-concat"             % "0.2.0")
 addSbtPlugin("io.github.irundaia"        % "sbt-sassify"            % "1.5.2")
 addSbtPlugin("org.scalastyle"            %% "scalastyle-sbt-plugin" % "1.0.0")
```

Comment:
This can be updated to `addSbtPlugin("com.beautiful-scala"       % "sbt-scalastyle"     % "1.5.1")`. Thanks